app/assets/stylesheets/modules/forms.scss

```diff
@@ -1,4 +1,13 @@
 .form-group {
+  a.btn:focus,
```

Comment:
Should these rules apply outside of the `.form-group` class or should they be global to any input/button regardless of where it shows up?